OgreMain/src/WIN32/OgreFileSystemLayer.cpp

```diff
@@ -122,6 +122,17 @@ namespace Ogre
         }
 #endif
 
+        if (char* val = getenv("OGRE_CONFIG_PATHS"))
```

Comment:
what is the use-case of specifying multiple paths here?

Reply:
I just wanted to be consistent with `mConfigPaths`. Don't care, though.

Reply:
my idea was that if you choose to override the default path resolution, you have a specific path in mind.

Use `OGRE_CONFIG_DIR` then - like the CMake variable:
https://ogrecave.github.io/ogre/api/latest/setup.html#cmake